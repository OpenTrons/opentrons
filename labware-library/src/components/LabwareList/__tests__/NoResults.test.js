// @flow
// NoResults tests
import * as React from 'react'
import { shallow } from 'enzyme'

import { NoResults } from '../NoResults'

describe('NoResults', () => {
  it('component renders', () => {
    expect(shallow(<NoResults />)).toMatchSnapshot()
  })
})
