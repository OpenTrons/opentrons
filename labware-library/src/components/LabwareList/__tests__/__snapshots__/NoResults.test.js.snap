// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoResults component renders 1`] = `
<p
  className="no_results"
>
  No results found
</p>
`;
